advocacy_docs/partner_docs/SIBVisionsGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Vision X'
+---
+
+Implementing VisionX with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components:
+
+- EDB Postgres Advanced Server
+- EDB Postgres Extended
+- VisionX low-code platform
+- Oracle database
+- Eclipse with VisionX ePlug 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance
+- A running EDB Postgres Extended instance
+- A running Oracle database instance
```

Comment:
EDB products should be one line.

A running EDB Postgres Advanced Server or EDB Postgres Extended Server